app/Rules/Patrimonio.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Uspdev\dadosUsp;
+
+class Patrimonio implements Rule
+{
+    /**
+     * Create a new rule instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        if ($value == '') {
```

Comment:
Essa linha não faz muito sentido, aliás, tá meio gambiarrada. O ideal seria validar como `required` ou `sometimes` quando o dado for enviado na request.

Reply:
Isso, vamos aceitar essa PR e já criei uma issue para consertar isso: 
https://github.com/uspdev/copaco/issues/177 #177

Reply:
Já me atribuí-a-mim-mesmo-me-lhe a issue :stuck_out_tongue: